autogpt_platform/autogpt_libs/autogpt_libs/auth/depends.py

```diff
@@ -35,3 +35,12 @@ def verify_user(payload: dict | None, admin_only: bool) -> User:
         raise fastapi.HTTPException(status_code=403, detail="Admin access required")
 
     return User.from_payload(payload)
+
+
+def get_user_id(payload: dict = fastapi.Depends(auth_middleware)) -> str:
+    user_id = payload.get("sub")
+    if not user_id:
+        raise fastapi.HTTPException(
+            status_code=401, detail="User ID not found in token"
+        )
+    return user_id
```

Comment:
why not (re)use this:
https://github.com/Significant-Gravitas/AutoGPT/blob/abf73e8d663487762ce7ac69b5cee73b7ff8f4e8/autogpt_platform/backend/backend/server/utils.py#L10-L11

Reply:
I didn’t notice it because the filename ‘utils’ didn’t hint at any auth-related stuff. And it looks like there are some differences in how they work.